instrumentation/redis/CHANGELOG.md

```diff
@@ -2,10 +2,16 @@
 
 ### Unreleased
 
+* ADDED: Updated API depedency for 1.0.0.rc1
 refactor: redis attribute utils [#760](https://github.com/open-telemetry/opentelemetry-ruby/pull/760)
 refactor: simplify redis attribute assignment [#758](https://github.com/open-telemetry/opentelemetry-ruby/pull/758)
 test: split redis instrumentation test [#754](https://github.com/open-telemetry/opentelemetry-ruby/pull/754)
 
+### v0.18.0 / 2021-05-21
+
+* ADDED: Option to obfuscate redis arguments
+* FIXED: Instrument Redis more thoroughly by patching Client#process.
+
```

Comment:
```suggestion
### v0.18.0 / 2021-05-21

* ADDED: Updated API depedency for 1.0.0.rc1
refactor: redis attribute utils [#760](https://github.com/open-telemetry/opentelemetry-ruby/pull/760)
refactor: simplify redis attribute assignment [#758](https://github.com/open-telemetry/opentelemetry-ruby/pull/758)
test: split redis instrumentation test [#754](https://github.com/open-telemetry/opentelemetry-ruby/pull/754)
* ADDED: Option to obfuscate redis arguments
* FIXED: Instrument Redis more thoroughly by patching Client#process.

```

Reply:
Thanks for catching that.  A lot of manual updates on this one.